engine/src/main/java/org/terasology/engine/rendering/logic/ChunkMeshComponent.java

```diff
@@ -25,4 +25,11 @@ public ChunkMeshComponent(ChunkMesh mesh, AABBf aabb) {
         this.mesh = mesh;
         this.aabb = aabb;
     }
+
+    @Override
+    public void copy(ChunkMeshComponent other) {
+        this.mesh = other.mesh; // TODO deep or shallow copy?
```

Comment:
we're assuming that we don't need to do deep-copies of assets. if at some point we find proof against this assumption, we'll have to have another look at this